storage-node/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 3.1.0
+
+- Adds connected query-node's state information to the `/status` endpoint.
```

Comment:
Could you also add information about the refactorization w.r.t. `QueryNodeApi` (single instance instead of multiple ones)

Reply:
[e545d72](https://github.com/Joystream/joystream/pull/4560/commits/e545d72a8d17c3f1f3c016cd2ef157b3935cc9a8)